### DISCUSSION TITLE
Page reloads after user script error

### DISCUSSION THREAD

**Discussion by andresclua** [swup/swup#917] [Help] [answered]
### What did you expect? 🧐

Randomly sometimes page transition does not work because website refresh.

### What actually happened? 😵‍💫

Page refresh when clicking

### Swup and plugin versions 📝

- swup:
"swup": "^4.1.0"
 "@swup/debug-plugin": "^4.0.1",
    "@swup/ga-plugin": "^2.0.0",
    "@swup/head-plugin": "^2.1.1",
    "@swup/js-plugin": "^3.1.1",
    "@swup/scripts-plugin": "^2.0.0",

### Relevant log output 🤓

```shell
Hey everyone,

I've been working on a few different projects, and I've encountered a bug that occurs when I transition from one page to another. The problem is that when the error appears, the page refreshes immediately, making it difficult to identify the issue.

To reproduce the bug, try navigating across pages with the console open.

Also, does anyone know if there is any documentation available for integrating the parallel plugin with SwupJsPlugin?
```

### URL to minimal reproduction 🔗

https://refactor-small-project--andrescluaportfolio.netlify.app/

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for related problems
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment, marked as answer]
You can configure your browser dev tools to preserve any console log output across page navigations. In Chrome, the option is called "Preserve log", not sure about other browsers.

The error you're seeing is `TypeError: Cannot read properties of null (reading 'clear') at ts.destroy at _astro/InfiniteMarque.9c44e3c5.js` — seems to be an issue with a marquee component used on the site.

**Reply by andresclua** [reply]
TSM @daun !

**Comment by daun** [comment]
Regarding parallel animations in JS, here's a previous discussion with example code: [Combining JS Plugin and Parallel Plugin](https://github.com/orgs/swup/discussions/799)

**Comment by daun** [comment]
A note on errors in user hooks: starting with version [4.5.1](https://github.com/swup/swup/releases/tag/4.5.1), swup will no longer force a hard reload when an uncaught error occurs in a user hook. Upgrading to 4.5.1 or later should solve your problem of not seeing the error as the navigation would be allowed to happen anyway, keeping the log entry in your console.

**Reply by andresclua** [reply]
tsm @daun ! - I was able to fix my issue - TSM TSM TSM
